client/a8c-for-agencies/sections/signup/signup-v2/components/signup-wrapper/style.scss

```diff
@@ -5,9 +5,14 @@
     display: flex;
     height: 90vh;
     background-color: var(--color-surface);
-    padding: 16px;
-    margin: 16px;
     border-radius: 16px; //stylelint-disable-line scales/radii
+	margin: -16px;
+
+
+	@include break-medium {
```

Comment:
Let's use break-small to target devices <600px, this targets upto 780px which is a tablet device.